doc/pmempool/pmempool-sync.1.md

```diff
@@ -75,6 +75,10 @@ be opened are recreated.=e=)
 
 ##### Available options: #####
 
+`-r, --redo-logs`
+
+: Read bad blocks from redo logs instead of checking them.
```

Comment:
This isn't crystal clear to me. I understand the intention, but I think we need to elaborate a little bit more about the effect of this option.